### ISSUE TITLE
Import Batch | Add Content | Content File | Browse... button - does not select my xml file

### ISSUE THREAD

**Issue by jfhencken** [concrete5/addon_migration_tool#19]
Migration Tool v.0.5.4 for C5.7+

I am unable to import an XML file which was created by the legacy export function. the legacy migration_tool worked just fine.
I have rebooted.  I have tried different browsers.  I have reinstalled the migration_tool.  Nothing.
The "Browse..." button opens a dialog window (Windows 7) and I select the XML file and press "Open" but nothing happens.  No error messages, nothing.
Would love to start migrating but with this issue I am stuck.
Please help.
-John

![file_upload_dialog](https://cloud.githubusercontent.com/assets/20561783/16994299/c720806a-4e74-11e6-89e1-4814f2c15e72.jpg)
![add_content_dialog](https://cloud.githubusercontent.com/assets/20561783/16994300/c723ad4e-4e74-11e6-9b90-8ab76c0bf613.jpg)

**Comment by jfhencken** [comment]
Turns out that the file is actually selected because it works when I click on "Add Content".
Am I correct in believing that the "Current File" input field should actually show the filename I just selected?

**Comment by aembler** [comment]
Yes, it should; this is a bug. No idea why it's doing that but it's definitely a bug (possibly with jQuery UI)

**Comment by jfhencken** [comment]
Just to follow up.  I decided to press the "Add Content" button anyway after selecting my XML file.  It worked!  The only issue is that the filename does not appear in the "Content File" field.
